hmrc-mongo-metrix-play-26/README.md

```diff
@@ -49,11 +49,9 @@ this information to graphite.
 
 ## Example construction
 ``` scala
-val mongoLockService = new MongoLockService {
-  override def mongoLockRepository: MongoLockRepository  = new MongoLockRepository()
-  override val lockId: String                            = "your-lock-id"
-  override val ttl: Duration                             = Duration(schedulerConfig.frequency().toMillis, TimeUnit.MILLISECONDS)
-}
+val lockId: String                            = "your-lock-id"
+val ttl: Duration                             = Duration(schedulerConfig.frequency().toMillis, TimeUnit.MILLISECONDS)
```

Comment:
Shouldn't this be just `schedulerConfig.frequency()` (since is already a Duration)
or even an example independent of other libraries e.g. `"1.minute"`